src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -3559,14 +3559,14 @@ void ObjectMgr::LoadPlayerInfo()
     {
         uint32 oldMSTime = getMSTime();
 
-        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() + 1, 0);
+        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() - 1, 0);
```

Comment:
Tested it. All Level xp were loaded.
The Table has 124 rows.

Row 0 is Header stuff.
So 123 rows with data.

Getting the row count and removing the Header will still leave 123 rows. Starting loading from Level 1 Not 0 (which doesnt exist) will cause full loading. 

But i can undo this Part